packages/plugin-e2e/tests/datasource/configEditor.spec.ts

```diff
@@ -2,14 +2,14 @@ import { expect, test } from '../../src';
 
 test('invalid credentials should return an error', async ({ createDataSourceConfigPage, page }) => {
   const configPage = await createDataSourceConfigPage({ type: 'grafana-googlesheets-datasource' });
-  await page.getByText('API Key', { exact: true }).click();
+  await page.getByText('API Key', { exact: true }).locator('..').click();
```

Comment:
A note on this change:
In a recent [change](https://github.com/grafana/grafana/pull/78010) in Grafana UI, the z-index for the radio button was changed. This means the radio itself is not clickable, so here I'm getting its parent. 

This highlights why one wants to use data-testid as selector instead. However, here I'm only installing a certain version of the Clock panel, so I cannot change its source code and add a proper selector for the radio button